### PR TITLE
Add schema registry connectivity checkt to camus etl

### PR DESCRIPTION
**Pull request by saulius**
Extracted common checking code to `camus-schema-registry` and reused in both in `camus-etl` and `camus-sweeper`.

@ljank @astrauka @vidma

### CODE REVIEW

File: camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/CamusJob.java [hunk 1]
```diff
@@ -94,7 +96,10 @@
 	public static final String KAFKA_HOST_URL = "kafka.host.url";
 	public static final String KAFKA_HOST_PORT = "kafka.host.port";
 	public static final String KAFKA_TIMEOUT_VALUE = "kafka.timeout.value";
-	public static final String LOG4J_CONFIGURATION = "log4j.configuration";
+  public static final String LOG4J_CONFIGURATION = "log4j.configuration";
```
Comment (astrauka) [inline camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/CamusJob.java]: alignment

File: camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/CamusJob.java [hunk 2]
```diff
@@ -233,7 +238,16 @@ public void run() throws Exception {
 		Job job = createJob(props);
 		if (getLog4jConfigure(job)) {
 			DOMConfigurator.configure("log4j.xml");
-		}
+    }
+
+    if (Boolean.parseBoolean(props.getProperty(SCHEMA_REGISTRY_CONNECTIVITY_CHECK))) {
```
Comment (astrauka) [inline camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/CamusJob.java]: code is ok, much java to parse boolean

Comment (astrauka) [inline camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/CamusJob.java]: why cannot we use booleans straight away?

Comment (saulius) [inline camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/CamusJob.java]: What do you mean exactly? https://docs.oracle.com/javase/7/docs/api/java/util/Properties.html#getProperty(java.lang.String) returns `String`s, need to cast to `Boolean` for the conditional.

File: camus-sweeper/pom.xml [hunk 3]
```diff
@@ -17,6 +17,10 @@
     </properties>
 
     <dependencies>
+      <dependency>
+        <groupId>com.linkedin.camus</groupId>
+        <artifactId>camus-schema-registry</artifactId>
```
Comment (ljank) [inline camus-sweeper/pom.xml]: why do we need this dependency?

Comment (saulius) [inline camus-sweeper/pom.xml]: To be able to reference `com.vinted.camus.schemaregistry.ConnectivityCheck` in `camus-sweeper`.

File: camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/CamusJob.java (hunk 2 above)
Comment (astrauka) [inline camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/CamusJob.java]: ok

**Comment by astrauka** [comment]
:+1:

**Pull request merged**